pkg/common/configs/configvalidator_test.go

```diff
@@ -1108,14 +1110,16 @@ func TestCheckLimits(t *testing.T) { //nolint:funlen
 				},
 				Limits: []Limit{
 					{
-						Limit:        "user-limit",
-						Users:        []string{"test-user"},
-						MaxResources: map[string]string{"memory": "100"},
+						Limit:           "user-limit",
+						Users:           []string{"test-user"},
+						MaxApplications: 1,
+						MaxResources:    map[string]string{"memory": "100"},
 					},
 					{
-						Limit:        "group-limit",
-						Groups:       []string{"test-group"},
-						MaxResources: map[string]string{"memory": "200"},
+						Limit:           "group-limit",
+						Groups:          []string{"test-group"},
+						MaxApplications: 1,
+						MaxResources:    map[string]string{"memory": "200"},
```

Comment:
Please ensure following cases has been covered:

1. When max app is 0, assert the error message
2. When max resource is memory: 0, cpu: 0 etc, assert the error message.
3. When max resource is  memory: 2, cpu: 0 etc, no errors should be thrown.
4. When limit doesn't have any either max app or max resource, assert the error message. No point in allowing the empty limit config.

Reply:
Cover them. Thank you.